### PR TITLE
#167164938 set up sass configuration

### PR DESCRIPTION
**Pull request by Rythae**
#### Pivotal Tracker Story

[ #167164938](https://www.pivotaltracker.com/story/show/167164938)

#### What does this PR do?
 set up sass configuration

#### Summary of Task
Screenshot 2019-08-27 at 1 39 46 PM

- install sass-loader and node-sass
- add sass configuration to webpack configuration file

 ### How can this be tested?
- git clone branch `chore/167164938/setup-sass`
- install dependencies
- run npm install
- start dev server npm run dev

### CODE REVIEW

File: webpack.config.production.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ module.exports = {
               sourceMap: true,
```
Comment (Nkemjiks) [inline webpack.config.production.js:33]: This file is missing the `test: /\.(css|scss)$/`. Implement the change you have on line 22 of the development config

Comment (Rythae) [reply webpack.config.production.js:33]: Alright. Thanks for the observation. I'm on it

**Review by Nkemjiks — APPROVED** [review]
Squash your commits together so i can merge

**Comment by Rythae** [comment]
> Squash your commits together so i can merge
Alright

**Pull request merged**